base/commands/serializer/serializer_it_test.go

```diff
@@ -0,0 +1,199 @@
+package serializer
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const generatedTestFilesDirectoryName = "generatedTestFiles"
+
+var (
+	allTypes,
+	nestedCompact,
+	allTypesSchema,
+	allTypesSchemaPath,
+	noNamespace,
+	noNamespaceNested,
+	noNamespaceSchemaPath,
+	noNamespaceSchema,
+	external,
+	externalSchemaPath,
+	externalSchema,
+	student,
+	classroom,
+	school,
+	schoolSchemaPath,
+	schoolSchema,
+	example1,
+	example2,
+	example3,
+	internalClassReferenceSchemaPath,
+	internalClassReference string
+)
+
+func init() {
+	_, filename, _, _ := runtime.Caller(0)
+	generationTestFilesDir := filepath.Join(filepath.Dir(filename), "testdata", "generationTestFiles")
```

Comment:
We shouldn't need to do this? Wouldn't `testdata/generationTestFiles` sufficient?